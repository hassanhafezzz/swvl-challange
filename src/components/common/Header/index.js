import React from 'react';
import classNames from 'classnames/bind';
import logo from '../../../img/logo.png';
import styles from './styles.module.css';

const cx = classNames.bind(styles);
export default function Header() {
  return (
    <header className={cx('header')}>
      <h1 className={cx('brand')}>
        <a href="/">
          <span className={cx('visually-hidden')}>swvl</span>
          <img src={logo} className={cx('logo')} alt="swvl logo" />
        </a>
      </h1>
    </header>
  );
}
