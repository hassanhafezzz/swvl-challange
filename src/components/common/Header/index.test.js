import React from 'react';
import { render } from '@testing-library/react';
import Header from './index';

test('header has logo', () => {
  const { getByAltText } = render(<Header />);
  const logo = getByAltText('swvl logo');
  expect(logo).toBeTruthy();
});

test('add snapshot for header ', () => {
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});
