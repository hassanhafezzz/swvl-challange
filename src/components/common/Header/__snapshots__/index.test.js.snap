// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add snapshot for header  1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <h1
      class="brand"
    >
      <a
        href="/"
      >
        <span
          class="visually-hidden"
        >
          swvl
        </span>
        <img
          alt="swvl logo"
          class="logo"
          src="logo.png"
        />
      </a>
    </h1>
  </header>
</DocumentFragment>
`;
